doc/source/guide_templates.rst

```diff
@@ -0,0 +1,36 @@
+Guide for Araali SW Template
+============================
+
+Araali baselines your application communication and presents them as identity-based links that can then be accepted as policy. This means no handwriting policies, everything is automatically discovered.
```

Comment:
check the documentation in google doc and pick up from there (response to customer)